data/resources.json

```diff
@@ -3,5 +3,77 @@
   "description": "The biggest Google tech conference in Ukraine carefully crafted for you by GDG community! All about Android, Web and Cloud from the world experts",
   "keywords": "event, gdg, gde, devfest, google, programming, android, chrome, polymer, developers, web, cloud, androiddev",
   "dates": "October 13-14, 2017",
-  "locationAddressShort": "Lviv, Ukraine"
+  "locationAddressShort": "Lviv, Ukraine",
+  "share": "Share",
+  "followOur": "Follow our",
+  "followUs": "Follow us",
+  "emailUs": "Email us",
+  "blog": "blog",
+  "partnershipResources": "Partnership Resources",
+  "eventResources": "Event Resources",
+  "footerRelBlock": [
```

Comment:
No, cause it's related to footer-rel.html element. Can be footerRelationsBlock, but I would leave it as it is.